app/classifier/tasks/survey/chooser.cjsx

```diff
@@ -2,6 +2,11 @@ React = require 'react'
 TriggeredModalForm = require 'modal-form/triggered'
 sortIntoColumns = require 'sort-into-columns'
 
+# key codes
+BACKSPACE = 8
```

Comment:
Pressing "Delete" on my Mac in Firefox makes the browser go back. Before it goes back it accurately deletes the survey task selection, which I can see when I click the "Get Started" button again, as it'll be where I left off (showing selections I still had, removed selection I pressed "Delete" to remove).